### PR TITLE
Namespaces

### PR DESCRIPTION
**Pull request by decentraliser**
- Simplify the namespace object sent to the view
- Dynamic expiration blockTime in view
- Simplify namespace display:
-- Remove list expansion
-- Contextualize actions (alias and extension)

image

### CODE REVIEW

File: src/components/namespace/NamespaceList.vue [hunk 1]
```diff
@@ -40,47 +40,38 @@
...
+                        : spawnExtendNamespace(iterNamespace)"
                     >
                       extend
                     </a>
                   </div>
                 </v-list-tile-sub-title>
                 <v-list-tile-sub-title>
                   <div class="monospaced">
-                    {{ ns.type }} {{ ns.alias }}
+                    {{ messages.namespaces['namespace alias type'][iterNamespace.aliasType] }}
```
Comment (evias) [inline src/components/namespace/NamespaceList.vue]: space in key ? This is better to be avoided.

File: src/components/namespace/NamespaceList.vue [hunk 2]
```diff
@@ -121,24 +120,43 @@ export default {
...
+      this.currentNamespaceEndHeight = namespace.endHeight;
       this.extendNamespace = true;
     },
+    spawnAliasTransaction(namespace) {
+      this.currentNamespaceName = namespace.name;
+      this.currentAliasType = namespace.aliasType;
+      this.currentAlias = namespace.aliasText;
+      this.aliasTransaction = true;
+    },
+    expirationText(namespace) {
```
Comment (evias) [inline src/components/namespace/NamespaceList.vue]: validator logic

File: src/locales/messages.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+// Temporary file till the translation module is installed
+
+export default {
```
Comment (evias) [inline src/locales/messages.js]: better to integrate PO system

Comment (decentraliser) [reply src/locales/messages.js]: This is a temporary file that will be adapted after i18n branch merge

File: src/components/namespace/NamespaceList.vue (hunk 1 above)
Comment (decentraliser) [reply src/components/namespace/NamespaceList.vue]: OK, I'll fix spaces in key in the next i18n PR

File: src/components/namespace/NamespaceList.vue (hunk 2 above)
Comment (decentraliser) [reply src/components/namespace/NamespaceList.vue]: I don't understand the validator logic issue here...
This component is listing namespaces, it has no input that alters data.
``spawnExtendNamespace`` and ``spawnAliasTransaction`` are here to show forms to craft transactions for the clicked namespace.

Maybe ``this.aliasTransaction = true`` is confusing? I could rename it to ``this.showAliasTransaction = true``;

**Pull request merged**